encoding/codecs/x264.md

```diff
@@ -25,7 +25,12 @@ Here's how we get a copy of the x264 encoder:
 
 ### Windows
 
-TODO
+Official Windows builds are available
+[here](https://download.videolan.org/pub/x264/binaries/win64/).
+For these examples,
+you'll want the latest version
+of the standard 8-bit build,
+which is the version without "10b" (for 10-bit) in the name.
```

Comment:
The binaries were unified some time ago. That’s the reason there are no 10-bit binaries after r2851. Depth is now controlled by `--output-depth`